recipes/piper/meta.yaml

```diff
@@ -26,9 +26,9 @@ requirements:
     - pandas
     - psutil
     - python >=3
-    - ubiquerg >=0.4.5
-    - yacman
-    - pipestat >=0.4.0
+    - ubiquerg >=0.8.0
```

Comment:
ubiquerg 0.8.0 has been updated 2 months ago - https://github.com/conda-forge/ubiquerg-feedstock/pull/10
But for some reason it is not yet available at https://anaconda.org/search?q=ubiquerg